### ISSUE TITLE
Remove GoogleCloudMessaging dependency for ios

### ISSUE THREAD

**Issue by CoDanny** [phonegap/phonegap-plugin-push#1419]
### Expected Behaviour

We use Amazon SNS for push delivery and would like to remove the dependency to GoogleCloudMessaging from iOS

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)

iOS

#### Cordova CLI version and cordova platform version

N/A

#### Plugin version

1.9.1

**Comment by macdonst** [comment]
@CoDanny I'm assuming you just don't want the GCM libs in your iOS app?

**Comment by CoDanny** [comment]
@macdonst that is correct. Since I'm not going to use them, it would be ideal to not have to include them

**Comment by macdonst** [comment]
@CoDanny okay, sounds good. I'd love to be able to do something like this but next step is 2.0.0 and FCM support. Then, everything else.

**Comment by CoDanny** [comment]
@macdonst Sounds good. Appreciated!
